extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -127,263 +138,466 @@ For example
+>         "POSITION": 0,
+>         "TEXCOORD_0": 1
+>       },
+>       "indices": 2,
+>       "material": 0,
+>       "extensions": {
+>         "EXT_mesh_features": {
+>           "propertyTextures": [0]
+>         }
+>       }
+>     }
+>   ]
+> }
+> ```
+
+
+A `propertyTexture` object extends the glTF [`textureInfo`](../../../../../specification/2.0/schema/textureInfo.schema.json) object. `texCoord` specifies a texture coordinate set in the referring primitive.
+
+The `properties` map specifies the texture channels providing data for available properties. An array of integer index values identifies channels, where multiple channels may be used only for fixed-length arrays of 2, 3, or 4 components. Channels of an `RGBA` texture are numbered 0–3 respectively, although specialized texture formats may allow additional channels. All values are stored in linear space.
+
+> **Example:** A property texture specifying that "thermalTemp" property values are stored in channel `R`, and "insulation" property values are stored in channel `G`, both indexed by UV coordinates in a `TEXCOORD_0` attribute.
+>
+> ```jsonc
+> // Root EXT_mesh_features extension:
+> {
+>   "propertyTextures": [{
+>     "class": "thermalSample",
+>     "index": 0,
+>     "texCoord": 0,
+>     "properties": {
+>       "thermalTemp": [0],
+>       "insulation": [1]
+>     }
+>   }]
+> }
 
 Texture filtering must be `9728` (NEAREST), `9729` (LINEAR), or undefined, for any texture object referenced as a property texture.
 
+*Defined in [propertyTexture.schema.json](./schema/propertyTexture.schema.json)*
+
 ## Binary Data Storage
 
-`EXT_mesh_features` imposes additional binary data alignment requirements on an asset, extending the 4-byte alignment in the core glTF specification:
+Feature properties are stored in a compact binary tabular format described in the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata), with each property table array occupying a glTF buffer view. `EXT_mesh_features` imposes additional binary data alignment requirements on an asset, further tightening the 4-byte alignments in the core glTF specification:
 
 - GLB-stored `JSON` chunk must be padded with trailing `Space` characters (`0x20`) to 8-byte boundary.
 - GLB-stored `BIN` chunk must be padded with trailing zeroes (`0x00`) to 8-byte boundary.
 
 As a result, byte length of the `BIN` chunk may be up to 7 bytes larger than JSON-defined `buffer.byteLength` to satisfy alignment requirements.
 
+## Optional vs. Required
+
+This extension is optional, meaning it should be placed in the `extensionsUsed` list, but not in the `extensionsRequired` list.
+
+## Schema
+
+* [gltf.EXT_mesh_features.schema.json](./schema/gltf.EXT_mesh_features.schema.json)
+* [primitive.EXT_mesh_features.schema.json](./schema/primitive.EXT_mesh_features.schema.json)
+* [node.EXT_mesh_features.schema.json](./schema/node.EXT_mesh_features.schema.json)
+
 ## Examples
```

Comment:
Meta: I like the breadth of examples here but I think it would be better as real glTF examples. Should this section be removed?

Reply:
For now, I'd leave this as is (I do like the table with diagrams). Though I made a note on our GH project board to consider making actual glTF files for these in the future.

If we do create samples for these, this table might be nice for a landing page?